doc/library/compile/function.txt

```diff
@@ -50,7 +50,7 @@ Reference
 
         Initialize attributes from arguments.
 
-.. class:: Param
```

Comment:
For the new arguments, I wrote a shorter version of the description in ```__init__```. Could you please check if that is right?